src/util/cache.ts

```diff
@@ -0,0 +1,10 @@
+/*
+ * Copyright (c) 2021, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+class Cache extends Map {}
+
+export default new Cache();
```

Comment:
Cache is not equal to a Map.  There needs to be a lot more to this class for it to be a proper cache.  This is only used by SfdcUrl right?  Can we move it there?